content/projects/gsoc/2019/project-ideas/machine-learning.adoc

```diff
@@ -0,0 +1,23 @@
+---
+layout: gsocprojectidea
+title: "Jenkins and Machine Learning Plugins for Data Science"
+goal: "Create a new plugin for integrating Jenkins with one of Machine Learning tools (e.g. Jupyter Python, TensorBoard, or Sacred)"
+category: Plugins
+year: 2019
+status: published
```

Comment:
Should it be draft? I agree it is detailed enough, but some discussion could be nice

Reply:
sure, draft first works for me :)